### ISSUE TITLE
hl-line mode vs org-modern tables

### ISSUE THREAD

**Issue by TheJJ** [minad/org-modern#175]
`hl-line-mode` uses an overlay to highlight the current line, while `org-modern` uses text properties to prettify org tables.
the `hl-line` face just sets the background color, but nevertheless org-modern's table lines get invisible in the current line.

any idea how can these made be compatible?

with `org-modern`:
![image](https://github.com/minad/org-modern/assets/673543/6aab1527-e9f7-4fc6-8cc3-3c1185d41956)

without `org-modern`:
![image](https://github.com/minad/org-modern/assets/673543/9fd7ed63-c053-4f8a-88d2-d753232c5d34)

**Comment by minad** [comment]
This problem is theme-dependent. For reference, it does not occur with Prot's Ef and Modus themes.

**Comment by TheJJ** [comment]
You're right, thanks!
Fixed it in my theme SFTtech/starlit-emacs@136bbc4fc4961c5b2cd0824eb0762e672322fbd1 :)

```diff
-`(hl-line ((,colorful (:inverse-video nil :background ,.current-line))))
+`(hl-line ((,colorful (:background ,.current-line))))
```
